src/app/components/Editor.tsx

```
@@ -114,300 +268,702 @@ const Editor: React.FC<EditorProps> = ({
+          "type",
+          "checked",
+        ], // Add type and checked attributes
       });
-      
+
       return (
-        <div 
-          className="prose prose-slate max-w-none p-6 bg-white/50 backdrop-blur-sm rounded-lg shadow-sm min-h-[75vh]"
+        <div
+          className="prose prose-slate min-h-[75vh] max-w-none rounded-lg bg-white/50 p-6 shadow-sm backdrop-blur-sm"
           dangerouslySetInnerHTML={{ __html: sanitizedHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)